MODULE.bazel

```diff
@@ -67,6 +67,7 @@ single_version_override(
     patch_strip = 1,
     patches = [
         "@com_google_protobuf//:Disable_bundle_install.patch",
+        "@com_google_protobuf//:Neverlink_jruby_jars.patch",
```

Comment:
Nit: Can we add a TODO to clean this up once https://github.com/bazel-contrib/rules_ruby/pull/226 is released?

Reply:
Ack; I'll add that into my PR that moves these patches into the `patches` directory